packages/meditrak-server/src/routes/utilities/SurveyResponseVariablesExtractor.js

```diff
@@ -0,0 +1,110 @@
+/**
+ * Tupaia MediTrak
+ * Copyright (c) 2020 Beyond Essential Systems Pty Ltd
+ */
+
+import { ValidationError } from '@tupaia/utils';
+
+/**
+ * A function that extracts `entityId` and `answers` from a row
+ *
+ * @callback ResponseExtractor
+ * @async
+ * @param {string[]} row
+ * @returns {{ entityId, answers }}
+ */
+
+export class SurveyResponseVariablesExtractor {
+  /**
+   * @param {ModelRegistry} models
+   */
+  constructor(models) {
+    this.models = models;
+  }
+
+  /**
+   * @public
+   * @param {Object<string, Array>} rowsBySurvey
+   * @param {string} userId
+   * @returns {Promise<{country: *, countryId: *}>}
+   */
```

Comment:
I think all the jsdocs in this file could be fixed :).

Reply:
I forgot to do so. My bad.